projects/burnr/src/App.tsx

```diff
@@ -30,17 +30,18 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const App: React.FunctionComponent<Props> = ({ className }: Props) => {
-	const [endpoint] = useLocalStorage('endpoint');
-	const [localStorageAccount, setLocalStorageAccount] = useLocalStorage(endpoint?.split('-')[0]?.toLowerCase());
+	const api = useApiCreate();
+	const [endpoint, useEndpoint] = useLocalStorage('endpoint');
+	if (!endpoint) useEndpoint('Polkadot-WsProvider');
+	const end = endpoint || 'Polkadot-WsProvider';
```

Comment:
Isn't that doing what the line above already did?
If `endpoint` is not defined use `Polkadot-WsProvider`?